<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouterKeygen</class>
 <widget class="QMainWindow" name="RouterKeygen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RouterKeygen</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitterVertical">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitterHorizontal">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="0">
          <widget class="QTabWidget" name="networkSplitter">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::North</enum>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="elideMode">
            <enum>Qt::ElideMiddle</enum>
           </property>
           <widget class="QWidget" name="supportedNetworks">
            <attribute name="title">
             <string>Supported</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QTableWidget" name="supportedNetworkslist">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>100</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>MAC</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Strength</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="unlikelyNetworks">
            <attribute name="title">
             <string>Unlikely Supported</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QTableWidget" name="unlikelyNetworkslist">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>100</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>MAC</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Strength</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="unsupportedNetworks">
            <attribute name="title">
             <string>Unsupported</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QTableWidget" name="unsupportedNetworkslist">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>MAC</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Strength</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="forceRefresh">
             <property name="text">
              <string>Force Scan (Note:requires root privileges)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="refreshScan">
             <property name="maximumSize">
              <size>
               <width>85</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="networkListLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Scanned Networks</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="manualCalc">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0">
          <widget class="QLabel" name="macLabel">
           <property name="text">
            <string>MAC Address:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="manualCalcLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Manual Calculation</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="ssidInput"/>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>300</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="calculateButton">
             <property name="text">
              <string>Calculate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="ssidLabel">
           <property name="text">
            <string>Network Name:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="macInput">
           <property name="inputMask">
            <string>HH:HH:HH:HH:HH:HH; </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QListWidget" name="passwordsList"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="passwordsLabel">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Calculated Passwords</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionDonate"/>
    <addaction name="actionDonate_Google_Play"/>
    <addaction name="separator"/>
    <addaction name="actionFeedback"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
   </widget>
   <addaction name="menuPreferences"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionDonate">
   <property name="text">
    <string>Donate (Paypal)</string>
   </property>
  </action>
  <action name="actionDonate_Google_Play">
   <property name="text">
    <string>Donate (Google Play)</string>
   </property>
  </action>
  <action name="actionRun_on_Start_up">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run on Start up</string>
   </property>
  </action>
  <action name="actionRun_in_the_background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run in the background</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About RouterKeygen</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>Feedback and Bugs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
